homeassistant/components/komfovent/const.py

```diff
@@ -0,0 +1,8 @@
+"""Constants for the Komfovent integration."""
+
+DOMAIN = "komfovent"
+MANUFACTURER = "Komfovent"
```

Comment:
is there anti-pattern to do with it? I'd much rather not have magic variables like this in random places and keep them all together? Happy to change if that's the standard here

Reply:
> is there anti-pattern to do with it?

No, it is just not needed. It can be helpful for re-use, but when it is used only once, it doesn't add anything. Expect that in the place used you don't actually see what it is.